src/containers/UserList/UserContextMenuActions.js

```diff
@@ -61,6 +63,14 @@ export const isUserContextActionAllowed = (model, action) => {
             return (
                 access.update && currentUser.authorities.has('F_REPLICATE_USER')
             )
+        case reset_password:
+            return (
+                access.update &&
+                (currentUser.authorities.has('F_USER_ADD') ||
+                    currentUser.authorities.has(
+                        'F_USER_ADD_WITHIN_MANAGED_GROUP'
+                    ))
+            )
```

Comment:
I believe `access.update` should be enough here.

My understanding is that the `access` property takes into account the current user's authorities, plus sharing settings (which is probably only relevant for metadata objects). However, I've just consulted the developer docs and was unable to confirm this.

Reply:
I implemented this based on what @jbee told were the necessary authorities. So I'll defer to him for the details.

Reply:
Generally @HendrikThePendric is right, but modifying user is special as some more checks are done where the mentioned two authorities `F_USER_ADD` and `F_USER_ADD_WITHIN_MANAGED_GROUP` are involved. While a user with `F_USER_ADD` definitely can modify a user with only `F_USER_ADD_WITHIN_MANAGED_GROUP` also needs to be allowed to manage at least one group the modified user is a member of.  

FYI: We are going to expose an endpoint to make this easier for you where you can check is the current user is allowed to modify another user.

Reply:
Good to know, thanks for the intel @jbee